model/host/db.go

```diff
@@ -267,16 +268,13 @@ func CountAllRunningDynamicHosts() (int, error) {
 	return num, errors.Wrap(err, "counting running dynamic hosts")
 }
 
-func CountStartedTaskHosts() (int, error) {
-	num, err := Count(db.Query(startedTaskHostsQuery("")))
+// CountIdleStartedTaskHosts returns the count of task hosts that are not
+// currently running a task.
```

Comment:
It seems worth mentioning that the count is also limited to hosts in a starting state.

Reply:
Good idea, added.